Examples/MAX32520/Hello_World_Cpp/README.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+A basic getting started program.
+
+This version of Hello_World prints an incrementing count to the console UART and toggles an LED once every 500 ms.
+
+## Software
+
+### Project Usage
+
+Universal instructions on building, flashing, and debugging this project can be found in the **[MSDK User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/)**.
+
+### Project-Specific Build Notes
+
+* This project comes pre-configured for the MAX78000EVKIT.  See [Board Support Packages](https://analog-devices-msdk.github.io/msdk/USERGUIDE/#board-support-packages) in the UG for instructions on changing the target board.
```

Comment:
Looks like the Project-Specific Build Notes got copied into each README

Reply:
Thanks, good catch

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/619/commits/50d5e14fe13ef79420406d963373257d9a647584